spec/features/visitors/email_confirmation_spec.rb

```diff
@@ -55,7 +55,12 @@
 
   context 'confirmed user is signed in and tries to confirm again' do
     it 'redirects the user to the profile' do
-      sign_up_and_2fa
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
```

Comment:
same comment as before. we only used it twice and every time I saw it I had to view the external file to know what was going on. seems easier to just have the steps in two places.